CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 Changes by Version
 ==================
 
-0.19.1 (unreleased)
+0.20.0 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- **BREAKING** - At least thriftrw 1.0 is now required.
```

Comment:
Some customers are implicitly picking up the wrong versions right now because they have a `thriftrw > foo` constraint in their requirements.txt, and they're seeing a fairly cryptic attribute error when they try to use it. Similarly, I think that when they upgrade to 0.20, if the requirements.txt is out of date, they might end up with thriftrw < 1.0, and get a similar message. I figure it's better to be safe and classify this as breaking with an explicit migration step to ensure they're on the right version.